src/pages/library/DataBrowserDetails.js

```diff
@@ -109,8 +116,43 @@ export const SidebarComponent = ({ dataObj, id }) => {
   const { access } = dataObj
   const [showRequestAccessModal, setShowRequestAccessModal] = useState(false)
   const [feedbackShowing, setFeedbackShowing] = useState(false)
+  const [datasetNotSupportedForExport, setDatasetNotSupportedForExport] = useState(false)
+  const [snapshotExportJobId, setSnapshotExportJobId] = useState()
+  const [tdrSnapshotPreparePolling, setTdrSnapshotPreparePolling] = useState(false)
   const sidebarButtonWidth = 230
 
+
+  const importDataToWorkspace = dataset => {
+    Utils.cond(
+      [isWorkspace(dataset), () => {
+        Ajax().Metrics.captureEvent(`${Events.catalogWorkspaceLink}:detailsView`, {
+          id,
+          title: dataObj['dct:title'],
+          type: 'workspace'
```

Comment:
I'd rather not get in the habit of including type in our event logs. We can always derive type from the catalog ID, and we don't want the UI looking at type at all if we can avoid it.

Reply:
I disagree that we don't want to include things that are derivable with the id - I think that if we want this to be readable by product, we want to ensure they have the information they commonly will need without needing to ask us to go into the DB for them.

Reply:
At some future point I hope we can remove the conditional code and handle both workspace and snapshot cases using a single code path, like we do for all other catalog API calls. At that point we'd have to decide if we want to keep the `isWorkspace` logic around for the purposes of event logging. This could be a case of planning for a future that never arrives, but I'd like to avoid that future decision by not including it now.

Maybe as a proxy for this value you could include the access URL itself? That would be OK I think.